skfda/inference/anova/anova_oneway.py

```diff
@@ -0,0 +1,290 @@
+import numpy as np
+from skfda.misc.metrics import norm_lp
+from skfda.representation import FData, FDataGrid, FDataBasis
+from skfda.datasets import make_gaussian_process
+
+
+def v_sample_stat(fd, weights, p=2):
+    """
+    Calculates a statistic that measures the variability between groups of
+    samples in a :class:`skfda.representation.grid.FDataGrid` object.
+
+    The statistic defined as below is calculated between all the samples in a
+    :class:`skfda.representation.grid.FDataGrid` object with a given set of
+    weights, and the desired :math:`L_p` norm.
+
+    Let :math:`\{f_i\}_{i=1}^k` be a set of samples in a FDataGrid object.
+    Let :math:`\{w_j\}_{j=1}^k` be a set of weights, where :math:`w_i` is
+    related to the sample :math:`f_i` for :math:`i=1,\dots,k`.
+    The statistic is defined as:
+
+    .. math::
+        V_n = \sum_{i<j}^kw_i\|f_i-f_j\|^p
```

Comment:
In LaTeX the double pipe symbol when using the math mode is \|
The output of that docstring shows a double pipe.

Reply:
OMG, all my life doing it wrong... However, you should still escape the escape character or make the docstring a raw docsting with r, so that no invalid escape sequence warnings are generated.

Reply:
Fixed